src/render-html/messageAsHtml.js

```diff
@@ -3,6 +3,8 @@ import type { FlagsState, ReactionType, RealmEmojiType } from '../types';
 import { shortTime } from '../utils/date';
 import messageTagsAsHtml from './messageTagsAsHtml';
 import messageReactionListAsHtml from './messageReactionListAsHtml';
+import base64 from 'base-64';
+import RNFetchBlob from 'react-native-fetch-blob';
```

Comment:
Importing base64 to convert data loaded by fetch into base64 but the approach failed
Later approach was moved to loading image by react-native-fetch-blob which has built in base64 conversion functionality.